doc/recipe/preprocessor.rst

```diff
@@ -1475,6 +1476,29 @@ Parameters:
     ``'nearest'``. There is no default.
 
 
+``extract_location``
+--------------------
+
+Extract a single point using a location name, with interpolation
+(either linear or nearest). This preprocessor extracts a single
+location point from a cube, according to the given interpolation
+scheme ``scheme``. The function retrieves the coordinates of the
+location and then calls the :func:`esmvalcore.preprocessor.extract_point`
+preprocessor. It can be used to locate cities and villages,
+but also mountains or other geographical locations.
+
+.. note::
+   Note that this function's geolocator application needs a
+   working internet connection.
+
+Parameters
+  * ``cube``: the input dataset cube to extract a point from.
+  * ``location``: the reference location. Examples: 'mount everest',
+    'romania', 'new york, usa'. Raises ValueError if none supplied.
+  * ``scheme`` : interpolation scheme. ``'linear'`` or ``'nearest'``.
+    There is no default, raises ValuError if none supplied.
```

Comment:
```suggestion
    There is no default, raises ValuError if none supplied.

See also :func:`esmvalcore.preprocessor.extract_location`.
```

Reply:
fixed off-site since I also had to fix ValuError -> ValueError (for the 68th time :rofl: )